### DISCUSSION TITLE
Are there any plans to add the Grindstone?

### DISCUSSION THREAD

**Discussion by Mikzoru** [Roadhog360/Et-Futurum-Requiem#195] [Ideas]
I see it's been suggested once before and that blocks are considered "boring suggestions" but I'm currently struggling to find any use for the quadruple chest's worth of enchanted low-tier weapons I got from looting a dungeon and really want something a bit more balanced or at least more interesting than simply transferring the enchantments onto books, so I was hoping that I might see the grindstone come to 1.7.10 soon.

**Comment by Roadhog360** [comment]
I have been interested in pointing it, however the model would be very complex, and I have a hard time understanding the code for GUIs in Minecraft. I actually do believe this block is underrated and really useful, more than people make it out to be.

**Reply by Mikzoru** [reply]
I really wouldn't mind a simpler placeholder model, or even just a simple cube! But not sure what to say about the GUIs... I hope you can get that worked out soon, but please don't exhaust or overwork yourself ; ;
